src/scenes/TaskDetailsView/index.tsx

```diff
@@ -55,17 +67,19 @@ function TaskDetailsView({ isAgentView }: TaskDetailsViewProps){
             </h2>
           </div>
           <div className="flex justify-end items-center gap-2">
-            {isAgentView && profile?.role === Profile.Role.HuddleAgent && (
-              <>
-              <Button>
-                Edit
-                <HiPencil className="ml-2" />
-              </Button>
-              <Button color="failure">
-                Delete
-                <HiTrash className="ml-2" />
-              </Button></>
-            )}
+
+              {isAgentView && profile?.role === Profile.Role.HuddleAgent && (
+                  <>
+                  <Button onClick={() => setIsEditModalVisible(true)}>
+                      Edit
+                      <HiPencil className="ml-2" />
+                  </Button>
+                  <Button color="failure">
+                      Delete
+                      <HiTrash className="ml-2" />
+                  </Button></>
+              )}
+
```

Comment:
```suggestion
              {isAgentView && profile?.role === Profile.Role.HuddleAgent && (
                  <>
                    <Button onClick={() => setIsEditModalVisible(true)}>
                        Edit
                        <HiPencil className="ml-2" />
                    </Button>
                    <Button color="failure">
                        Delete
                        <HiTrash className="ml-2" />
                    </Button>
                  </>
              )}
```
Not that important but better to have them properly formatted

Reply:
fixed